src/ext/oblivc/obliv_bits.c

```diff
@@ -929,8 +926,12 @@ void setupYaoProtocol(ProtocolDesc* pd,bool halfgates)
   else ypd->recver.recver=NULL;
 
   dhRandomInit();
+
+  //TODO: why is this ECB?
```

Comment:
I think I answered this in an email. Either remove or describe briefly why this is necessary.